manim/cli/render/commands.py

```diff
@@ -136,4 +138,40 @@ def __repr__(self):
             except Exception:
                 console.print_exception()
 
+    if config.notify_outdated_version:
+        manim_info_url = "https://pypi.org/pypi/manim/json"
+        warn_prompt = "Cannot check if latest release of manim is installed"
+        req_info = {}
+
+        try:
+            req_info = requests.get(manim_info_url, timeout=3)
+            req_info.raise_for_status()
+        except requests.exceptions.HTTPError:
+            logger.warning(f"HTTP Error: {warn_prompt}")
+            raise SystemExit
+        except requests.exceptions.ConnectionError:
+            logger.warning(f"Connection Error: {warn_prompt}")
+            raise SystemExit
+        except requests.exceptions.Timeout:
+            logger.warning(f"Timed Out: {warn_prompt}")
+            raise SystemExit
+        except Exception:
+            logger.warning(f"Something went wrong: {warn_prompt}")
+            raise SystemExit
+
+        try:
+            releases = req_info.json()["releases"].keys()
+
+            if max(releases) != __version__:
+                console.print(
+                    f"You are using manim version [red]v{__version__}[/red], but version [green]v{max(releases)}[/green] is available."
+                )
+                console.print(
+                    "You should consider upgrading via [yellow]pip install -U manim[/yellow]"
+                )
+        except json.JSONDecodeError:
+            logger.warning(warn_prompt)
+            logger.warning(f"Error decoding JSON from {manim_info_url}")
+            raise SystemExit
```

Comment:
Again, doing exit or raise, this feature shouldn't make Manim exit with an error code.

Reply:
Also, it should use `logger.warning` rather it should use `logger.debug` because the user doesn't need to know that this failed.